src/components/AddItem/AddItem.component.jsx

```diff
@@ -28,7 +28,7 @@ const AddItem = () => {
   //To add the item to the database
   const addNewItemValue = event => {
     event.preventDefault();
-    firebase.dataBase
+    firebase.db
```

Comment:
Hi @brewswain, @stef-codes, and @daredtech! Do you think there is value using naming conventions that more descriptive over concise? However, it may just be my own personal preference 😀 .

Reply:
I agree, dataBase is more descriptive